pages/[id].js

```diff
@@ -1,81 +1,19 @@
-import { plants } from "@/assets/plants";
+import PlantDetails from "@/components/PlantDetails";
 import { useRouter } from "next/router";
-import Image from "next/image";
-import Link from "next/link";
-import styled from "styled-components";
-
-const lightNeedIcon = {
-  "Full Sun": "☀️☀️☀️☀️",
-  "Partial Shade": "☀️☀️☀️",
-  Shade: "☀️☀️",
-  "Full Shade": "☀️",
-};
-
-const waterNeedIcon = {
-  Low: "💧",
-  Medium: "💧💧",
-  High: "💧💧💧",
-};
-
-const seasonIcons = {
-  Spring: "🌱",
-  Summer: "🐝",
-  Fall: "🍂",
-  Winter: "❄️",
-};
-
-export default function PlantDetails() {
+import useLocalStorageState from "use-local-storage-state";
+export default function PlantDetailsPage() {
```

Comment:
```suggestion
export default function PlantDetailsPage({plants}) {
```

Reply:
Could you explain please why you added the prop `plants` here?